Configuration/AlCa/python/autoCond.py

```diff
@@ -24,23 +24,23 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run2
     'run2_mc_pa'                   :    '131X_mcRun2_pA_v2',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'                    :    '133X_dataRun2_v1',
+    'run2_data'                    :    '133X_dataRun2_v2',
     # GlobalTag for Run2 data 2018B relvals only: HEM-15-16 fail
-    'run2_data_HEfail'             :    '133X_dataRun2_HEfail_v1',
+    'run2_data_HEfail'             :    '133X_dataRun2_HEfail_v2',
```

Comment:
There was an error of mine while naming the updated DB: I wrote `133X_dataRun2_Hefail_v2` instead of `133X_dataRun2_HEfail_v2.` This originates the error in the IBs for (e.g.) wf 136.8642.

I'm going to fix it in the DB, and there should be no need to fix the PR after that!

Reply:
Now 133X_dataRun2_HEfail_v2 exists in the IB, identical to 133X_dataRun2_Hefail_v2: https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/133X_dataRun2_Hefail_v2/133X_dataRun2_HEfail_v2

I expect that the IB error in 136.8642 will disappear while testing next CMSSW_14_0_X_2023-11-18-1100